thunder/torch/__init__.py

```diff
@@ -802,6 +802,26 @@ def randn_like(
     return randn(a.shape, dtype=dtype, device=device)
 
 
+@torchsymbol(is_method=False, id="torch.normal")
+def normal(
+    mean: TensorLike,
```

Comment:
Looking at the [PyTorch documentation](https://pytorch.org/docs/stable/generated/torch.normal.html), `mean` could also be a `Number`, and in that case the output would have the shape of `std`.  It would be good to ensure all variations of shapes of `mean` and `std` are included in the test cases.